Jenkinsfile

```diff
@@ -92,63 +144,117 @@ pipeline {
     }
     stage('Build and deploy preview') {
       when {
-        branch 'PR-*'
+        allOf {
+          branch 'PR-*'
+          not {
+            environment name: 'DRY_RUN', value: 'true'
```

Comment:
nit: would be nice to move this to a `isDryRun()` if possible (not sure it'll work in these declarative blocks

Reply:
Don't think we can use it here...